internal/host/hostcommands/install_cmd_test.go

```diff
@@ -72,7 +72,7 @@ var _ = Describe("installcmd", func() {
 		mockVersions = versions.NewMockHandler(ctrl)
 		mockRelease = oc.NewMockRelease(ctrl)
 		installCmd = NewInstallCmd(common.GetTestLog(), db, mockValidator, mockRelease, instructionConfig, mockEvents, mockVersions, true, true)
-		cluster = createClusterInDb(db, models.ClusterHighAvailabilityModeFull)
+		cluster = createClusterInDb(db, 3)
```

Comment:
It would be great if you could use the variable `common.MinMasterHostsNeededForInstallationInHaMode` everywhere in the tests you change where the "3" number is just a default HA cluster, like here